src/app/updater.js

```diff
@@ -77,21 +79,44 @@
                 self.updateData = updateData;
                 return true;
             }
+            if (App.settings.UpdateSeed) {
+                client.add(updateData.UpdateUrl, { path: os.tmpdir() }, function (torrent) {
+                    torrent.on('error', function (err) {
+                        win.debug('ERROR' + err.message);
+                    });
+                    torrent.on('done', function () {
+                        win.debug('Seeding the Current Update!');
```

Comment:
so it looks like seed() is to create a new torrent, i'm not sure how seed control works...